e2e-tests/portal/cypress/integration/journeys/checker/cover-end-date/deal.js

```diff
@@ -0,0 +1,517 @@
+const { mandatoryCriteria } = require('../../../../../../e2e-fixtures');
+const CONSTANTS = require('../../../../fixtures/constants');
+const { nowPlusDays } = require('../../../../support/utils/dateFuncs');
+
+const date = new Date();
+const datePlusDay = nowPlusDays(1);
+
+const deal = {
```

Comment:
do we need this file? or can we inherit the properties from other fixture file instead?

Reply:
done - inherited